WebhookNotificationActivity/client.ts

```diff
@@ -0,0 +1,37 @@
+import { HttpsUrl } from "io-functions-commons/dist/generated/definitions/HttpsUrl";
+import {
+  ApiHeaderJson,
+  createFetchRequestForApi,
+  TypeofApiCall
+} from "italia-ts-commons/lib/requests";
+import * as r from "italia-ts-commons/lib/requests";
+import { ProblemJson } from "italia-ts-commons/lib/responses";
+import { Notification } from "../generated/notifications/Notification";
+import { notifyDefaultDecoder } from "../generated/notifications/requestTypes";
+import { SuccessResponse } from "../generated/notifications/SuccessResponse";
+
+export type WebhookNotifyT = r.IPostApiRequestType<
+  { readonly notification?: Notification; readonly webhookEndpoint: HttpsUrl },
```

Comment:
why is `notification` optional? what's the use case for an empty notification?

Reply:
The generated requestType (from io-utils) is:

```ts
// Request type definition
export type NotifyT = r.IPostApiRequestType<
  { readonly notification?: Notification },
  "Content-Type",
  never,
  | r.IResponseType<200, SuccessResponse>
  | r.IResponseType<400, ProblemJson>
  | r.IResponseType<401, undefined>
  | r.IResponseType<500, ProblemJson>
>;
```

probably because this parameter is not required
https://github.com/pagopa/io-backend/blob/master/api_notifications.yaml#L20

Reply:
it looks like it's [actually required](https://github.com/pagopa/io-backend/blob/master/src/controllers/notificationController.ts#L49), we should fix the specs

Reply:
fixed in io-backend